test/e2e/page-objects/pages/dialog/snap-install.ts

```diff
@@ -80,16 +85,8 @@ class SnapInstall {
     console.log(
       'Clicking on the scroll button and then clicking the confirm button',
     );
-    await this.driver.waitUntil(
-      async () => {
-        await this.driver.clickElementSafe(this.snapInstallScrollArea);
-        const isEnabled = await this.driver.findClickableElement(
-          this.nextPageButton,
-        );
-        return isEnabled;
-      },
-      { timeout: veryLargeDelayMs, interval: 100 },
-    );
```

Comment:
all this is not needed anymore, we just check the confirmation page is loaded, by looking for an element, and then we can trigger the scroll button